gradio/routes.py

```diff
@@ -253,6 +328,33 @@ def _cancel_asyncio_tasks(self):
             task.cancel()
         self._asyncio_tasks = []
 
+    @staticmethod
+    def start_node_server():
+        if GRADIO_DEV_MODE:
+            return
+        if App.node_path is None or App.force_spa or App.node_process is not None:
+            return
+        try:
+            App.node_process = subprocess.Popen([App.node_path, SSR_APP_PATH])
```

Comment:
We should set `App.node_process = None` initially, otherwise its a missing attribute if this doesn't launch correctly